app/assets/javascripts/contacts.js

```diff
@@ -72,8 +85,86 @@ var processContacts = function(contacts, status, xhr) {
   });
 };
 
-var showContactInForm = function(position) {
+var showEmptyContactList = function() {
+  $(".contacts").append(
+    "<li data-role='contact'>No contacts yet</li>"
+  );
+};
+
+var displayContacts = function() {
+  if (anyContacts()) {
+    showContactList();
+  }
+  else {
+    showEmptyContactList();
+  }
+};
+
+var processContacts = function(contacts) {
+  storedContacts = contacts;
```

Comment:
Ah. This seems confusing. I might move this to a function so you have something like "setContactList(contacts)". Not sure though, do whatever feels right.